integ/README.md

```diff
@@ -79,3 +79,40 @@ variables. Supported options include:
 You can also pass [gocheck](http://labix.org/gocheck) options to the
 test runner. Particularly useful is `-check.vv` which will output logs
 from the operator and Flink pods to help debugging test failures.
+
+### Minikube Setup
+
+Ideally we'd use k8s 1.16 to match the deployed k8s version, however, this
+is non-trivial due to cgroup configurations. Instead, we will use a version
+that is compatible with v1beta1 CRD's which corresponds to <1.22. CRD's v1
+is only available with client >=1.16, however, the client used here is 1.14
+and the upgrade is non-trivial.
+
+
+1. Install Dependencies
+   Run dep ensure -vendor-only
+
+3. Start minikube
+   minikube start --kubernetes-version=v1.20.15
+
+4. Proxy minikube
+   kubectl proxy --port 8001 &
+
+5. Set up test app images and operator image
+   integ/setup.sh
+
+8. Set the following for the Go test:
+   Package path: github.com/lyft/flinkk8soperator/integ
+   Env: INTEGRATION=true;OPERATOR_IMAGE=flinkk8soperator:local;RUN_DIRECT=true
+   Program Args: -timeout 40m -check.vv IntegTest
```

Comment:
I removed this when attempting to target a single test. No luck there for now so leaving as is since this is what was there for the integration test prior. In the future we'll figure out how to target a single test as I'll likely write an integration test for fallback without savepoint